packages/sysweb3-utils/src/getContract.ts

```diff
@@ -0,0 +1,51 @@
+import Web3 from 'web3';
+
+import { getErc21Abi, getErc55Abi } from './contracts';
+
+export const getContract = async (
+  contractAddress: string,
+  networkUrl: string
+) => {
+  const provider = new Web3.providers.HttpProvider(networkUrl);
+
+  const web3Provider = new Web3();
+
+  web3Provider.setProvider(provider);
+
+  const [abi721, abi1155] = [getErc21Abi(), getErc55Abi()];
+
+  const abisInfos = [
```

Comment:
Add the base interface for erc-20:
Erc20 base interface: 0x36372b07

Reply:
Fixed